tools/cldr-apps/src/org/unicode/cldr/web/OutputFileManager.java

```diff
@@ -635,9 +654,8 @@ private boolean verifyParentChildSameDirectory(Writer out, File vxmlDir)
                     }
                     CLDRLocale parLoc = childLoc.getParent();
                     if (parLoc != null) {
-                        // String parentName = fileName.replaceFirst("_[a-zA-Z]+\\.xml$", "\\.xml");
                         String parentName = parLoc.toString() + XML_SUFFIX;
-                        if (!childName.equals(parentName) && !"root.xml".equals(parentName)) {
+                        if (!childName.equals(parentName) && !"en.xml".equals(parentName) && !"root.xml".equals(parentName)) {
```

Comment:
Not sure about this. Can you explain more what is happening?

Reply:
When I first made a change to prevent the generation of en.xml and root.xml, I got this:

Verification failure, child without parent
Child, present: /Users/tbishop/Documents/WenlinDocs/Organizations/Unicode/CLDR_job/vetdata-2020-08-06T17-49-10-961Z/vxml/common/main/en_PR.xml
Parent, absent: /Users/tbishop/Documents/WenlinDocs/Organizations/Unicode/CLDR_job/vetdata-2020-08-06T17-49-10-961Z/vxml/common/main/en.xml

This change prevents that spurious failure, making it OK to have en_PR.xml without en.xml.